### ISSUE TITLE
DS2422+ Can't get IP & no any HDD activity 7.2 64561

### ISSUE THREAD

**Issue by Edwardpge** [pocopico/tinycore-redpill#320]
No Internet activity and no hard drive looks like kernal panic, Works fine with 7.1.1 4962 but not with 7.2 64561. VT-1000 architecture have some kernal issue,,

**Comment by pocopico** [comment]
Yes, i've noticed also on my testing, one question, do you boot using EFI or LEGACY ?

**Comment by ner00** [comment]
Same issue, seems to almost have started happening within the past 12 hours; was working fine last night as I did multiple tests.

**Comment by ner00** [comment]
Got the latest release and it seemed to fix everything for me, plus e1000 driver is now supported.
The web builder is really cool but still has the odd issue of sometimes having an incomplete cmdline when booting. Second time worked fine, not really sure what triggers that problem.

**Comment by Edwardpge** [comment]
> Yes, i've noticed also on my testing, one question, do you boot using EFI or LEGACY ?

I tried UEFI

**Comment by Edwardpge** [comment]
> Got the latest release and it seemed to fix everything for me, plus e1000 driver is now supported with latest 7.2. The web builder is really cool but still has the odd issue of sometimes having an incomplete cmdline when booting. Second time worked fine, not really sure what triggers that problem.

are you sure about e1000 NIC driver? are they triggering the panic?

**Comment by pocopico** [comment]
Can you please stop at tcrpfriend at the countdown by pressing ctrl-c  and perform the following

vi /mnt/tcrp/user_config.json and check that your USB/SATA line contains the word withefi 

if you get a message that booting EFI with no EFI can cause issues then the withefi command line option is missing and is causing issues (nops and KP) , specifically with ds2422 if you boot using EFI.

I've tested that and when you add withefi option issues are gone.

**Comment by Edwardpge** [comment]
> Can you please stop at tcrpfriend at the countdown by pressing ctrl-c  and perform the following
> 
> vi /mnt/tcrp/user_config.json and check that your USB/SATA line contains the word withefi 
> 
> if you get a message that booting EFI with no EFI can cause issues then the withefi command line option is missing and is causing issues (nops and KP) , specifically with ds2422 if you boot using EFI.
> 
> I've tested that and when you add withefi option issues are gone.

Can't boot with efi invalid command line error

**Comment by ner00** [comment]
> are you sure about e1000 NIC driver? are they triggering the panic?

Previously e1000 would work on DSM 7.1.1 but would not on 7.2, I can't say if it was caused by kernel panic because the only thing I noticed is the Ethernet card never getting recognized. I'd need to switch e1000 to e1000e between 7.1.1 and 7.2. With the latest bootloader it works fine with e1000 on both versions without the need to swap them in-between upgrades.

**Comment by Edwardpge** [comment]
> > are you sure about e1000 NIC driver? are they triggering the panic?
> 
> Previously e1000 would work on DSM 7.1.1 but would not on 7.2, I can't say if it was caused by kernel panic because the only thing I noticed is the Ethernet card never getting recognized. I'd need to switch e1000 to e1000e between 7.1.1 and 7.2. With the latest bootloader it works fine with e1000 on both versions without the need to swap them in-between upgrades.

I have tried with e1000e NIC nothing changed even after removing the NIC. Working fine with 7.1.1

**Comment by huehuw** [comment]
I also ran into HDD issues with the new build. After compiling it fails to detect any of my hard drives not mater what I try.

**Comment by pocopico** [comment]
> I also ran into HDD issues with the new build. After compiling it fails to detect any of my hard drives not mater what I try.

-Which Platform and what settings did you use ?
-What kind of drives (SATA/SAS/AHCI) ?

**Comment by ChenC2022** [comment]
> I also ran into HDD issues with the new build. After compiling it fails to detect any of my hard drives not mater what I try.

me too. my PVE v7.4-3, host board J4125.

**Comment by pocopico** [comment]
Please try again with latest release v0.9.4.9b
